index.html

```diff
@@ -293,29 +293,56 @@ <h3>Expected Behavior</h3>
     
   <section id="Reporting">
     <h2>Reporting</h2>
-    <p>If you are concerned about your immediate safety, contact <a href="https://en.wikipedia.org/wiki/List_of_emergency_telephone_numbers">local law enforcement</a>.
-      For a face to face event you may need to contact venue staff for assistance contacting them.</p>
-    <p>You may also contact a W3C ombud who will assist you: <a href="mailto:ombuds@w3.org">ombuds@w3.org</a>.</p>
+    <p>If you are concerned about your immediate safety, contact 
+      <a href="https://en.wikipedia.org/wiki/List_of_emergency_telephone_numbers">local law enforcement</a>.
+      For a face to face event Chairs and venue staff should provide any necessary assistance contacting them.
```

Comment:
As a Chair I am only occasionally the _host_ of a meeting. If I am in a foreign country I feel no more able to assist with contacting law enforcement than anyone else. Propose that we introduce the concept of "host" for face to face meetings and expect them to help here, as well as venue staff.

Editorial nit: it took me three goes reading this to realise that the "them" at the end of the sentence refers to "local law enforcement" and not to "Chairs and venue staff". Perhaps there's a way to edit this so it is easier to parse. I don't have any immediate suggestions though :-(

Reply:
noted, we will pay attention to this as we work on chairs training. And, we will work on antecedents.